lookout/style/format/tests/test_analyzer.py

```diff
@@ -81,6 +82,9 @@ def test_train(self):
         model2 = FormatAnalyzer.train(self.ptr, config, datastub)
         self.assertEqual(model1["javascript"].rules, model2["javascript"].rules)
         self.assertGreater(len(model1["javascript"]), 10)
+        # Check that model can be saved without problems and then load back
+        with TemporaryFile(prefix="analyzer_model-") as f:
+            model2.save(f)
```

Comment:
we should probably load the model back to make sure everything is fine (and to sync with the comment). We can maybe share code with test_model to do so :)

Reply:
+1